index.html

```diff
@@ -160,16 +160,16 @@ <h3 id="noharm" data-export="" data-dfn-type="dfn">
   </h3>
   <p>
 When we are adding a feature or technology to the web,
-we will work to prevent or mitigate any harm it might cause society or groups,
+we will prevent or mitigate any harm it might cause society or groups,
```

Comment:
This statement does not seem possible.

Reply:
This is true, but then so much of this document is aspirational to the point of being borderline unrealistic, maybe this is OK.  But the point @cwilso makes is a good one.  This is an area in which there are no clean absolutes.  Change will hurt people.  Not changing will too.

How about we look at a different framing?  See #126 for my attempt to turn the hedging here into the core message.